### PR TITLE
initial-release-notes-6.6.9.0

### PR DESCRIPTION
**Pull request by nam4am**

### CODE REVIEW

**Comment by silverDuy** [comment]
The title and the commit should simply be "v6.6.9.0"

File: src/6.6/unreleased.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+nav:
+  title: v6.6.?.?
```
Comment (silverDuy) [inline src/6.6/unreleased.md:3]: v6.6.9.0

File: src/6.6/unreleased.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+meta:
+  date: "????-??-??"
+---
+
+# Release notes Shopware 6.6.?.?
```
Comment (silverDuy) [inline src/6.6/unreleased.md:8]: 6.6.9.0

File: src/6.6/unreleased.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Fixed bugs
+
+## Credits
+
+Thanks to all diligent friends for helping us make Shopware better and better with each pull request!
+
+## More resources
+
+* [Detailed diff on Github](https://github.com/shopware/shopware/compare/v6.6.6.1...v6.6.7.0) to the former version
```
Comment (silverDuy) [inline src/6.6/unreleased.md:47]: Please correct the link to git diff on Github.

File: src/6.6/unreleased.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* [Changelog on GitHub](https://github.com/shopware/shopware/blob/v6.6.7.0/CHANGELOG.md) for this version.
```
Comment (silverDuy) [inline src/6.6/unreleased.md:48]: this one too 😄

File: src/6.6/6.6.9.0.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
+---
+nav:
+  title: v6.6.9.0
+meta:
+  date: 2024-11-08
```
Comment (bojanrajh) [inline src/6.6/6.6.9.0.md:5]:
```suggestion
  date: 2024-12-03
```

File: src/6.6/6.6.9.0.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+Added the ability to view search explanations in the browser's network tab when using the preview search API, integrated with the Preview search UI.
+
+### Exclude products from sitemap
+
+Added the option to exclude hidden products from the sitemap.
+
+### Accessibility improvements
+
+A bypass mechanism was implemented to enhance site navigation by providing ARIA landmarks, internal skip links, and headlines for site areas. 
+Additionally, accessible names for <iframe> elements were added, and methods such as collapsible sections and labeled landmarks were recommended to improve usability for screen reader and keyboard-only users.
```
Comment (bojanrajh) [inline src/6.6/6.6.9.0.md:60]:
```suggestion
Additionally, accessible names for `<iframe>` elements were added, and methods such as collapsible sections and labeled landmarks were recommended to improve usability for screen reader and keyboard-only users.
```

**Review by bojanrajh — APPROVED** [review]

**Pull request merged**